tensorflow/matmul.py

```diff
@@ -17,6 +17,14 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+# ML framework used.
+_ARGS_ML_FRAMEWORK = os.getenv('ML_FRAMEWORK', 'tensorflow')
+print("ML_FRAMEWORK set to %s" % _ARGS_ML_FRAMEWORK, file=sys.stderr)
+
+# Performance Indicator used.
+_ARGS_PI_NAME = os.getenv('PI', 'matmul')
+print("PI set to %s" % _ARGS_PI_NAME, file=sys.stderr)
```

Comment:
I think these can be directly hard-coded into the script. There's no point why they should be parametrized.